addons/l10n_in/views/report_invoice.xml

```diff
@@ -2,7 +2,24 @@
 <odoo>
     <template id="l10n_in_report_invoice_document_inherit" inherit_id="account.report_invoice_document">
 
-
+        <xpath expr="//span[@t-field='o.partner_id.vat']" position="replace">
+            <t t-if="o.company_id.country_id.code == 'IN'" class="mt16">
+                <span t-field="o.l10n_in_gstin"/>
+            </t>
+            <t t-else="">
+                <span t-field="o.partner_id.vat"/>
+            </t>
+        </xpath>
```

Comment:
Please check if this is compatible with other inheritances.  I think there is a div around that might be better to replace

Reply:
Remove replace, and use t-if
Thank you